frontend/src/my_team/MyTeamView.jsx

```diff
@@ -4,25 +4,35 @@ import { useState, useContext } from "react";
 import { UserContext } from "../UserContext.js";
```

Comment:
Demo for selecting playing styles feature

https://github.com/user-attachments/assets/59da375d-d51c-4955-b865-7f372509e57f